docs/gwcs/points_to_wcs.rst

```diff
@@ -0,0 +1,51 @@
+.. _user_api:
+
+Fitting a WCS to input pixels & sky positions
+=============================================
+
+Suppose we have an image where we have centroid positions for a handful of sources, and we have matched these 
+positions to an external catalog to obtain (RA, Dec). If this data is missing or has inaccurate WCS information,
+it is useful to fit or re-fit a GWCS object with this matched list of coordinate pairs to be able to transform
+between pixel and sky. 
+
+This example shows how to use the wcs_from_point tool to fit a WCS to a matched set of 
+pixel and sky positions.  Along with arrays of the (x,y) pixel position in the image and the matched sky coordinates,
+the fiducial point for the projection must be supplied as a SkyCoord object. Additionally,
+the projection type must be specified from the available projections in `~astropy.modeling.projections.projcode`.
+
+Geometric distortion can also be fit to the input coordinates - the distortion type (2D polynomial, chebyshev, legendre) and 
+the degree can be supplied to fit this component of the model.
+
+The following example will show how to fit a WCS, including a 4th degree 2D polynomial, to a set of input pixel positions of 
+sources in an image and their corresponding positions on the sky obtained from a catalog. 
+
+.. doctest-skip::
+   
+  >>> from gwcs import wcs_from_point
+  >>> import numpy as np
+  >>> from astropy.coordinates import SkyCoord
+  
+  The matched set of input pixels and sky coordinates are passed in as tuples of arrays, matched by index. 
+  The xy array is a tuple (x coordinates, y coordinates), and world_coordinates is (RA, Dec).
+  
```

Comment:
The example below fails because numpy is not imported. You may need other imports below.